package/CHANGELOG

```diff
@@ -71,6 +71,9 @@ Enhancements
     keyword (Issue #3605)
 
 Changes
+  * The minimm Python, NumPy, and SciPy versions (following NEP29) are now 3.8, 
```

Comment:
I edited CHANGELOG to address my comment...

Reply:
Thanks!